libraries/lib-wave-track/WaveTrack.h

```diff
@@ -455,6 +455,74 @@ class WAVE_TRACK_API WaveTrack final
       // contiguous range.
       sampleCount* pNumWithinClips = nullptr) const override;
 
+   /*!
+    * @brief Gets as many samples as it can, but no more than `2 *
+    * numSideSamples + 1`, centered around `t`. Reads nothing if
+    * `GetClipAtTime(t) == nullptr`. Useful to acess samples across clip
+    * boundaries, as it spreads the read to adjacent clips, i.e., not separated
+    * by silence from clip at `t`.
+    *
+    * @return The begin and end indices of the samples in the buffer where
+    * samples could actually be copied.
+    */
+   std::pair<size_t, size_t> GetFloatsCenteredAroundTime(
+      double t, size_t iChannel, float* buffer, size_t numSideSamples,
+      bool mayThrow) const;
+
+   /*!
+    * @brief Similar to GetFloatsCenteredAroundTime, but reads only in one
+    * direction. @see GetFloatsCenteredAroundTime
+    *
+    * @return The number of samples actually copied.
+    */
+   size_t GetFloatsFromTime(
```

Comment:
Clarify that the resulting samples are not reversed when direction is backward.

Reply:
Moved those to `private` and made comments a bit more precise.